elasticsearch_collectd.py

```diff
@@ -572,6 +915,21 @@ def init_stats():
     if ES_VERSION.startswith("2."):
         NODE_STATS_CUR.update(NODE_STATS_ES_2)
 
+    # Attempt to parse the major, minor, and revision
+    major = ES_VERSION.split('.')[0]
+    minor = ES_VERSION.split('.')[1]
+    revision = ES_VERSION.split('.')[2]
+
+    # Iterate over deprecation lists and remove any keys that were deprecated
+    # prior to the current version
+    for dep in DEPRECATED_NODE_STATS:
```

Comment:
move this into its own function + the version parsing

remove_deprecated_metrics(es_version)